src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OpenSslAsnFormatter.cs

```diff
@@ -48,11 +48,11 @@ protected override string FormatNative(Oid oid, byte[] rawData, bool multiLine)
                         return null;
                     }
 
-                    int printLen = Interop.libcrypto.GetMemoryBioSize(bio);
+                    int printLen = Interop.Crypto.GetMemoryBioSize(bio);
 
                     // Account for the null terminator that it'll want to write.
                     StringBuilder builder = new StringBuilder(printLen + 1);
-                    Interop.libcrypto.BIO_gets(bio, builder, builder.Capacity);
+                    Interop.Crypto.BioGets(bio, builder, builder.Capacity);
```

Comment:
That's a good question :wink:  With the current marshaling implementation (which I think we should look at fixing), I'm having trouble coming up with a good argument for why you'd ever want to use one, at least as [Out].  For [In], it might be a little bit more reasonable, but even there the current marshaling implementation has some costs that you could likely workaround using something other than StringBuilder.